example.typ

```diff
@@ -1,4 +1,4 @@
-#import "@local/bytefield:0.0.3": *
+#import "bytefield.typ" : *
```

Comment:
could not compile without changing this

Reply:
Yeah don't mind. I added a symlink inside the local typst package folder to the repo folder to check if it works as a package.